src/nvim/fold.c

```diff
@@ -545,11 +543,9 @@ static int checkCloseRec(garray_T *gap, linenr_T lnum, int level)
 }
 
 // foldCreateAllowed() {{{2
-/*
- * Return TRUE if it's allowed to manually create or delete a fold.
- * Give an error message and return FALSE if not.
- */
-int foldManualAllowed(int create)
+/// @return  TRUE if it's allowed to manually create or delete a fold.
+///          Give an error message and return FALSE if not.
```

Comment:
nit: Not sure how this will show up in UIs/LSP clients. If there is nothing else to say I would just keep it in the main description instead of moving the main description to `@return`.